src/nvim/ex_cmds.lua

```diff
@@ -15,3288 +15,3277 @@ local COUNT      =    0x400
 local NOTRLCOM   =    0x800
 local ZEROR      =   0x1000
 local USECTRLV   =   0x2000
-local NOTADR     =   0x4000
-local EDITCMD    =   0x8000
-local BUFNAME    =  0x10000
-local BUFUNL     =  0x20000
-local ARGOPT     =  0x40000
-local SBOXOK     =  0x80000
-local CMDWIN     = 0x100000
-local MODIFY     = 0x200000
-local EXFLAGS    = 0x400000
-local RESTRICT   = 0x800000
+local EDITCMD    =   0x4000
+local BUFNAME    =   0x8000
+local BUFUNL     =  0x10000
+local ARGOPT     =  0x20000
+local SBOXOK     =  0x40000
+local CMDWIN     =  0x80000
+local MODIFY     = 0x100000
+local EXFLAGS    = 0x200000
+local RESTRICT   = 0x400000
```

Comment:
https://github.com/neovim/neovim/pull/11996 removed restricted mode so `RESTRICT` is now dead code. Maybe comment it out and update affected ex commands.